GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeView.swift

```diff
@@ -63,7 +63,9 @@ extension CallVisualizer {
             }
         }
 
-        let refreshButton = UIButton()
+        let refreshButton = UIButton().make { button in
+            button.accessibilityIdentifier = "visitorCode_refresh_button"
```

Comment:
`Minor`. To be honest, mix of camel and snake cases looks a bit strange.

Reply:
I saw that all our `accessibilityIdentifiers` were written as a snake, so I followed the pattern.

Reply:
I meant mix of two cases `"visitorCode_refresh_button"`, not only snake `"visitor_code_refresh_button"`
But I guess that is the logical separation of the levels of UI elements. Lets leave as is.

Reply:
I got it, my bad. Fixed [f3a4586](https://github.com/salemove/ios-sdk-widgets/pull/467/commits/f3a45861a88d6038850e6290b288c8f35c25ad58)